appsignals-tests/contract-tests/src/test/java/software/amazon/opentelemetry/appsignals/test/jdbc/operationtests/DBOperation.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.opentelemetry.appsignals.test.jdbc.operationtests;
+
+public enum DBOperation {
```

Comment:
IMO too much abstraction, just create 2 tests, one for select and another for create. No need to be fancy, it only makes things more complex

Reply:
IMO having all in the same class was just too much and actually complex to read, took me sometime to understand as well.

what would you think as alternative, also notice this way will enable us to have more operations to test and even more DB Engines 👀

Happy to hear other alternatives that more middle ground here.